graylog2-server/src/main/java/org/graylog2/configuration/ElasticsearchClientConfiguration.java

```diff
@@ -59,6 +59,9 @@
     @Parameter(value = "elasticsearch_discovery_frequency", validator = PositiveDurationValidator.class)
     Duration discoveryFrequency = Duration.seconds(30L);
 
+    @Parameter(value = "elasticsearch_discovery_default_scheme")
```

Comment:
Could you implement a `Validator<String>` for this? I think we should only allow `http` and `https` values for this setting. You can find an example for a validator here: 

https://github.com/Graylog2/graylog2-server/blob/cb33786987e845aed42956bed67a335c2b91471c/graylog2-server/src/main/java/org/graylog2/configuration/validators/ListOfURIsWithHostAndSchemeValidator.java#L26-L37

Reply:
I will check it later!